content/docs/contributing.md

```diff
@@ -8,29 +8,37 @@ Participation in Open Source projects is highly encouraged, and the Open Source
 
 ### Summary
 
-- Upstream contributions are highly encouraged
+We highly encourage contributions to existing open source projects. There are some requirements before you start patching, and guides to help you figure out the right path:
+
+- Patches generally don't require any review from the Open Source Program Office
```

Comment:
Are there cases where OSPO needs to review? I think the idea is having Saucers follow the policies, but if OSPO needs to review patches things could get slow.

Reply:
There are OSPOs that require a review for patches (e.g. Google for patches over 100 lines of code). However with this sentence we state that no review from the OSPO is required for any patches as we don't have the capacities to do that.

Reply:
👍